tests/simapp/providers.go

```diff
@@ -1,307 +0,0 @@
-package simapp
```

Comment:
Moving this out of our app and into a separate directory since we no longer run anything in process.